R/store.R

```diff
@@ -26,8 +26,9 @@ file_store <- R6::R6Class(
     ## TODO: bulk get, with overwrite control
     get = function(hash, dst) {
       src <- self$filename(hash)
-      if (!file.exists(src)) {
-        stop(sprintf("Hash '%s' not found in store", hash))
+      if (any(!file.exists(src))) {
+        stop(sprintf("Hash '%s' not found in store",
+                     hash[which(!file.exists(src))]))
```

Comment:
no need for which here

```suggestion
                     hash[!file.exists(src)]))
```